### PR TITLE
docs: update readme to indicate the port binding

### PR DESCRIPTION
**Pull request by felipementel**
In order to make it easier for developers who want to run SonarQube in a container, I think it is worth indicating the port binding when running the docker run command.

Our exemple are using the tag 10.6.0-community

### CODE REVIEW

**Comment by andreracz** [comment]
Super useful information.

File: sonarqube/README.md [hunk 1]
```diff
@@ -102,6 +102,14 @@ To run a cluster with the Data Center Edition, please refer to Installing SonarQ
 
 ## Configuration
 
+### Port binding
+
+By default, the image will use port 9000
```
Comment (whalelines) [inline sonarqube/README.md:107]: Perhaps something like "By default, the server running within the container will listen on port 9000. You can expose the container port 9000 to the host port 9000 with the `-p 9000:9000` argument to `docker run`:" would be more clear?

**Review by whalelines — COMMENTED** [review]
Minor suggestion

**Comment by tianon** [comment]
A few notes:
- this should be in `content.md`, not `README.md` (the latter is autogenerated, per the comment at the top of it)
- it should use `%%IMAGE%%` for image references
- `10.6.0` is a _really_ specific image reference that should probably be somewhat relaxed (either 10.6 or 10)
- cc `sonarqube` image maintainers: @carminevassallo @jCOTINEAU @davividal

File: sonarqube/README.md (hunk 1 above)
Comment (felipementel) [reply sonarqube/README.md:107]: I will update the text, like your suggestion.

I also will ajust the pull request like @tianon indicated.

**Comment by felipementel** [comment]
@tianon and @whalelines 
Can you verify my updates?

**Comment by jCOTINEAU** [comment]
all good on our side; thanks a lot for taking the time to improve our documentation.

File: sonarqube/content.md [hunk 2]
```diff
@@ -37,6 +37,16 @@ To run a cluster with the Data Center Edition, please refer to Installing SonarQ
 
 ## Configuration
 
+### Port binding
+
+By default, the server running within the container will listen on port 9000. You can expose the container port 9000 to the host port 9000 with the `-p 9000:9000` argument to `docker container run`, like the command below:
+
+```console
+docker container run --name sonarqube-custom -p 9000:9000 sonarqube:10.6-community
```
Comment (tianon) [inline sonarqube/content.md:45]:
```suggestion
docker run --name sonarqube-custom -p 9000:9000 %%IMAGE%%:10.6-community
```

File: sonarqube/content.md [hunk 3]
```diff
@@ -49,9 +59,9 @@ Set up a database by following the "Installing the Database" section of https://
 
 We recommend creating volumes for the following directories:
 
--	`/opt/sonarqube/data`: data files, such as the embedded H2 database and Elasticsearch indexes
--	`/opt/sonarqube/logs`: contains SonarQube logs about access, web process, CE process, Elasticsearch logs
--	`/opt/sonarqube/extensions`: for 3rd party plugins
+- `/opt/sonarqube/data`: data files, such as the embedded H2 database and Elasticsearch indexes
+- `/opt/sonarqube/logs`: contains SonarQube logs about access, web process, CE process, Elasticsearch logs
+- `/opt/sonarqube/extensions`: for 3rd party plugins
```
Comment (tianon) [inline sonarqube/content.md:64]:
```suggestion
-	`/opt/sonarqube/data`: data files, such as the embedded H2 database and Elasticsearch indexes
-	`/opt/sonarqube/logs`: contains SonarQube logs about access, web process, CE process, Elasticsearch logs
-	`/opt/sonarqube/extensions`: for 3rd party plugins
```
(these were intentional from `markdownfmt`)

File: sonarqube/content.md [hunk 4]
```diff
@@ -66,15 +76,15 @@ For upgrade instructions, see Upgrading from the Docker Image on the [Upgrade th
 In some environments, it may make more sense to prepare a custom image containing your configuration. A `Dockerfile` to achieve this may be as simple as:
 
 ```dockerfile
-FROM sonarqube:8.9-community
+FROM sonarqube:10.6-community
```
Comment (tianon) [inline sonarqube/content.md:79]:
```suggestion
FROM %%IMAGE%%:10.6-community
```

While we're here, we should fix this one too. :smile:

File: sonarqube/content.md [hunk 5]
```diff
@@ -66,15 +76,15 @@ For upgrade instructions, see Upgrading from the Docker Image on the [Upgrade th
...
 COPY sonar-custom-plugin-1.0.jar /opt/sonarqube/extensions/
 ```
 
 You could then build and try the image with something like:
 
 ```console
 $ docker build --tag=sonarqube-custom .
-$ docker run -ti sonarqube-custom
+$ docker container run -ti sonarqube-custom
```
Comment (tianon) [inline sonarqube/content.md:87]:
```suggestion
$ docker run -ti sonarqube-custom
```

File: sonarqube/content.md [hunk 6]
```diff
@@ -37,6 +37,16 @@ To run a cluster with the Data Center Edition, please refer to Installing SonarQ
 
 ## Configuration
 
+### Port binding
+
+By default, the server running within the container will listen on port 9000. You can expose the container port 9000 to the host port 9000 with the `-p 9000:9000` argument to `docker container run`, like the command below:
```
Comment (tianon) [inline sonarqube/content.md:42]:
```suggestion
By default, the server running within the container will listen on port 9000. You can expose the container port 9000 to the host port 9000 with the `-p 9000:9000` argument to `docker run`, like the command below:
```

**Review by tianon — APPROVED** [review]

**Comment by felipementel** [comment]
@tianon @jCOTINEAU @alvdavi 

thank you so much!

**Pull request merged**